src/ripple/app/misc/TxQ.h

```diff
@@ -306,7 +306,7 @@ class TxQ
 
     /** Return the next sequence that would go in the TxQ for an account. */
     SeqProxy
-    nextQueuableSeq(std::shared_ptr<SLE const> const& sleAccount) const;
+    nextQueuableSeq(std::optional<AcctRootRd const> const& acctRoot) const;
```

Comment:
This interface also shows a problem with the implementation of the wrappers. `AcctRootRd` is _really_ just an `AcctRoot` that is `const`, but instead of the standard C++ `AcctRoot const` we need to use a special `AcctRootRd` class, and I guess we may as well mark that `const` too.

I *know* this is a hard problem - I'm not criticizing. I'm fine with the current implementation, although I wish we could solve this in a reasonable way (we did explore a `const_cast` solution at one point, but I understand why we didn't pursue that - that has it's own set of trade-offs).